ROADMAP.md

```diff
@@ -0,0 +1,31 @@
+# Style Guides Roadmap
+
+> tl;dr The goal of this document is to document the priority items for style guide development over the next 3-6 months as well as key activities designed to grow awareness and adoption of the Style Guides at STScI.
+
+## Planned releases
+
+### v1.0 release (proposed release date: 3rd April 2020)
+
+Initial release of STScI Style Guides including:
+
+- Accessibility guide
+- GitHub repositories guide
+- Jupyter Notebooks guide
+- Python (naming, coding standards, testing etc.) guide
+- Python packaging guide
+- Security guide
+- Guidelines for how to advertise compliance with STScI Style Guides (https://github.com/spacetelescope/style-guides/issues/102)
+
+### v1.1 release (release date June 2020)
+
+- Merge/reconcile [INS standards for JWST community software](https://github.com/spacetelescope/ins-jwst-community-software) with STScI Style Guides.
```

Comment:
/ cc @bourque for 👀

Reply:
This date *should* be fine.  Within the next two months, @shanosborne and I will finish documenting all of the standards for the INS JWST community software tools, and will be meeting with the JWST mission office to make sure we are all on the same page moving forward.  So this is to say -- after that meeting, we should know how feasible a June 2020 release will be.